app/admin/projects.rb

```diff
@@ -86,6 +88,9 @@
       row 'Approved?' do ad.is_approved end
       row :created_at
       row :updated_at
+      row :twitter_url do |proj|
+        link_to proj.twitter, proj.twitter, target: "blank" if proj.twitter?
```

Comment:
Right now the link href is set to proj.twitter. You'll need to insert an `"https://twitter.com/" + ` into the second argument to get this going where you'd expect.

This is the last, last thing. Probably. :)